notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<ul><li><strong>Adding a quick one line description as the following might be helpful:</strong></li></ul><blockquote>A function to actual and demand position and velocity of TMA azimuth with respect to time. It includes vertical lines to indicate two different event and plot for interpolated acceleration.</blockquote><p>or to make it shorter,</p><blockquote class="ql-indent-1">Same as <code>plot_az </code>but for elevation.</blockquote><ul><li><strong>Moving this function close to </strong><code><strong>plot_az</strong></code><strong> might be helpful to organize this.</strong></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>